services/snapshots/service.go

```diff
@@ -255,6 +255,29 @@ func (s *service) Usage(ctx context.Context, ur *snapshotsapi.UsageRequest) (*sn
 	return fromUsage(usage), nil
 }
 
+type cleaner interface {
```

Comment:
this should be defined in snapshots/snapshotter.go ?
And how does it relate to Close()?

Reply:
I was following the principle of define where it is used, but with it in the API and used in multiple places, probably best to add it and document in `snapshots/snapshotter.go`

Reply:
seems also defined in `metadata/snapshot.go`